### DISCUSSION TITLE
Unable to find emitter for node type

### DISCUSSION THREAD

**Discussion by itslyn** [AvaloniaUI/Avalonia#13781] [Q&A] [answered]
I keep getting an error in Visual Studio. However, when I compile and run my program, it works perfectly fine including the user control that has the error. But, when I'm just working in Visual Studio, the error is causing me to not be able to use my designer. When I try to rebuild, it still has the error. I've also restarted Visual Studio several times. How can I fix this error? 
![image](https://github.com/AvaloniaUI/Avalonia/assets/115854752/44a639c7-ad8d-4594-8628-49c897af9011)

**Comment by timunie** [comment]
Does it happen on a blank new UserControl as well?

**Reply by itslyn** [reply]
No, it doesn't. A blank new UserControl doesn't have any errors and the designer loads it fine.
![image](https://github.com/AvaloniaUI/Avalonia/assets/115854752/c80cbf5e-f8aa-4222-bbfd-240323ffb3f9)
It is only this specific xaml file in the original post that has the problem with the designer.

**Reply by timunie** [reply]
that means you will have to find the actual root cause in your UserControl. Not much we can do here. You may want to try nightly if that has better error messages here but that is all I can offer rn.

**Reply by itslyn** [reply, marked as answer]
I figured it out! 
It was the width and height line for the UserControl. I had it as this:
![image](https://github.com/AvaloniaUI/Avalonia/assets/115854752/5dff352d-6981-499c-99ec-416dac7df414)
But, once I changed it to:
![image](https://github.com/AvaloniaUI/Avalonia/assets/115854752/9fac6170-7248-44e2-a7b9-86cf350d91ca)
It worked and the designer doesn't give any errors anymore and displays it properly. I'm new to this so didn't realize there was a difference but now I know.
